index.js

```diff
@@ -77,7 +77,7 @@ function start(swaggerFile, targetDir, port, hostname, openBrowser) {
     });
   });
 
-  watch(targetDir, {recursive: true}, function(eventType, name) {
+  chokidar.watch(targetDir).on('all', function(eventType, name) {
```

Comment:
The 'all' parameter will call the callback for every file present in the folder being watched with an 'add' eventType. This is problematic as we only want it to run for 'change' eventTypes. If you have a lot of files in the directory you're watching, the callback fires for every file simply existing on startup instead of just on change.
![2020-06-06-185145_732x308_scrot](https://user-images.githubusercontent.com/6024271/83945248-22d81980-a827-11ea-87c2-743a8050e35b.png)

Reply:
@moon0326  Can you make the above correction and merge this in? Chokidar seems like a [good choice](https://github.com/paulmillr/chokidar#why) and is very popular. I don't want to let this PR stagnate because of the above bug.

Reply:
The change has been merged and released as 2.1.9. Thank you @wand2016  and @louisnow !

Reply:
@moon0326 I don't see the change on master yet.

The line currently incorrectly says 

`chokidar.watch(targetDir).on('all', function(eventType, name) {`

when it should be

 `chokidar.watch(targetDir).on('change', function(eventType, name) {`

Reply:
Opened a PR #52 @moon0326